# Logfile created on 2013-08-09 15:36:18 -0400 by logger.rb/31641
2013-08-09 15:36:18 -0400 DEBUG: Connecting to Mongo at: 
2013-08-09 15:36:18 -0400 DEBUG: 54.242.195.20
2013-08-09 15:36:18 -0400 DEBUG: 27017
2013-08-09 15:36:57 -0400 DEBUG: Connecting to Mongo at: 
2013-08-09 15:36:57 -0400 DEBUG: 54.242.195.20
2013-08-09 15:36:57 -0400 DEBUG: 27017
2013-08-09 15:37:49 -0400 DEBUG: Connecting to Mongo at: 
2013-08-09 15:37:49 -0400 DEBUG: 54.242.195.20
2013-08-09 15:37:49 -0400 DEBUG: 27017
2013-08-09 15:38:19 -0400 ERROR: Cannot connect to Mongo
2013-08-09 15:38:19 -0400 DEBUG: ++++++++++++++++++++++++++
2013-08-09 15:38:19 -0400 DEBUG: /Users/plee/Code/external_api/config/settings.yml
2013-08-09 15:38:19 -0400 DEBUG: http://localservices.carecloud.local:3000/
2013-08-09 15:38:19 -0400 DEBUG: {"api_internal_svc_url"=>"http://localservices.carecloud.local:3000/", "api_internal_doc_srv_upld_url"=>"http://uploads-dev.carecloud.local/", "mirth_outbound_svc_url"=>"http://10.0.1.163", "mirth_private_key"=>"thisstringshouldbeprivate", "memcache_servers"=>"cc-bl-mcache00-dev.carecloud.local:11211", "mongo_server"=>"54.242.195.20", "mongo_port"=>27017, "lab_user"=>"spectra_test", "lab_pass"=>"MLhYSFoptX58EsSrdAX1iyb3cuPiJMud"}
2013-08-09 15:38:19 -0400 INFO: API-Service (Development) launched
2013-08-09 15:40:47 -0400 DEBUG: passed in token: AQIC5wM2LY4Sfcx/m25JBOSilhzVpnxeur2nHTgzLPyCS3Y=@AAJTSQACMDMAAlNLAAoxOTY0MzUyMDc3AAJTMQACMDE=#
2013-08-09 15:40:47 -0400 DEBUG: cache key: business-entity-AQIC5wM2LY4Sfcx/m25JBOSilhzVpnxeur2nHTgzLPyCS3Y=@AAJTSQACMDMAAlNLAAoxOTY0MzUyMDc3AAJTMQACMDE=#
2013-08-09 15:40:49 -0400 ERROR: cannot reach cache store
2013-08-09 15:40:49 -0400 DEBUG: business entity not found in cache, making the call
2013-08-09 15:40:49 -0400 DEBUG: url for business entity list: http://localservices.carecloud.local:3000/business_entities/list_by_user.json?list_type=list&token=AQIC5wM2LY4Sfcx%2Fm25JBOSilhzVpnxeur2nHTgzLPyCS3Y%3D%40AAJTSQACMDMAAlNLAAoxOTY0MzUyMDc3AAJTMQACMDE%3D%23
2013-08-09 15:40:53 -0400 ERROR: GET http://localservices.carecloud.local:3000/business_entities/list_by_user.json?list_type=list&token=AQIC5wM2LY4Sfcx%2Fm25JBOSilhzVpnxeur2nHTgzLPyCS3Y%3D%40AAJTSQACMDMAAlNLAAoxOTY0MzUyMDc3AAJTMQACMDE%3D%23 FAILED - Connection refused - connect(2)
2013-08-09 15:40:53 -0400 WARN: ----127.0.0.1 "GET /v2/appointment/listbyid/2ae8b08d-5d41-40e5-b068-41803fc689c4?authentication=AQIC5wM2LY4Sfcx%2Fm25JBOSilhzVpnxeur2nHTgzLPyCS3Y%3D%40AAJTSQACMDMAAlNLAAoxOTY0MzUyMDc3AAJTMQACMDE%3D%23" - 500 {"error":"Internal error while calling method. Please try again later"} - 5940 ms
2013-08-09 15:41:05 -0400 DEBUG: passed in token: AQIC5wM2LY4SfcwVdqRaz6bcq4j7zP0hAxEZCHypVI1vmFM=@AAJTSQACMDMAAlNLAAotNTkxMTY2MTczAAJTMQACMDE=#
2013-08-09 15:41:05 -0400 DEBUG: cache key: business-entity-AQIC5wM2LY4SfcwVdqRaz6bcq4j7zP0hAxEZCHypVI1vmFM=@AAJTSQACMDMAAlNLAAotNTkxMTY2MTczAAJTMQACMDE=#
2013-08-09 15:41:06 -0400 DEBUG: business entity not found in cache, making the call
2013-08-09 15:41:06 -0400 DEBUG: url for business entity list: http://localservices.carecloud.local:3000/business_entities/list_by_user.json?list_type=list&token=AQIC5wM2LY4SfcwVdqRaz6bcq4j7zP0hAxEZCHypVI1vmFM%3D%40AAJTSQACMDMAAlNLAAotNTkxMTY2MTczAAJTMQACMDE%3D%23
2013-08-09 15:41:11 -0400 ERROR: GET http://localservices.carecloud.local:3000/business_entities/list_by_user.json?list_type=list&token=AQIC5wM2LY4SfcwVdqRaz6bcq4j7zP0hAxEZCHypVI1vmFM%3D%40AAJTSQACMDMAAlNLAAotNTkxMTY2MTczAAJTMQACMDE%3D%23 FAILED - Connection refused - connect(2)
2013-08-09 15:41:11 -0400 WARN: ----127.0.0.1 "GET /v2/appointment/listbyid/2ae8b08d-5d41-40e5-b068-41803fc689c4?authentication=AQIC5wM2LY4SfcwVdqRaz6bcq4j7zP0hAxEZCHypVI1vmFM%3D%40AAJTSQACMDMAAlNLAAotNTkxMTY2MTczAAJTMQACMDE%3D%23" - 500 {"error":"Internal error while calling method. Please try again later"} - 5623 ms
2013-08-09 15:42:20 -0400 DEBUG: passed in token: AQIC5wM2LY4SfcwlqDjXF/GJDJpHPNR8Pwgf0mrg29JcGI8=@AAJTSQACMDMAAlNLAAotMTgzNjUwMDcwAAJTMQACMDE=#
2013-08-09 15:42:20 -0400 DEBUG: cache key: business-entity-AQIC5wM2LY4SfcwlqDjXF/GJDJpHPNR8Pwgf0mrg29JcGI8=@AAJTSQACMDMAAlNLAAotMTgzNjUwMDcwAAJTMQACMDE=#
2013-08-09 15:42:20 -0400 DEBUG: business entity not found in cache, making the call
2013-08-09 15:42:20 -0400 DEBUG: url for business entity list: http://localservices.carecloud.local:3000/business_entities/list_by_user.json?list_type=list&token=AQIC5wM2LY4SfcwlqDjXF%2FGJDJpHPNR8Pwgf0mrg29JcGI8%3D%40AAJTSQACMDMAAlNLAAotMTgzNjUwMDcwAAJTMQACMDE%3D%23
2013-08-09 15:42:25 -0400 ERROR: GET http://localservices.carecloud.local:3000/business_entities/list_by_user.json?list_type=list&token=AQIC5wM2LY4SfcwlqDjXF%2FGJDJpHPNR8Pwgf0mrg29JcGI8%3D%40AAJTSQACMDMAAlNLAAotMTgzNjUwMDcwAAJTMQACMDE%3D%23 FAILED - Connection refused - connect(2)
2013-08-09 15:42:25 -0400 WARN: ----127.0.0.1 "GET /v2/appointment/listbyid/2ae8b08d-5d41-40e5-b068-41803fc689c4?authentication=AQIC5wM2LY4SfcwlqDjXF%2FGJDJpHPNR8Pwgf0mrg29JcGI8%3D%40AAJTSQACMDMAAlNLAAotMTgzNjUwMDcwAAJTMQACMDE%3D%23" - 500 {"error":"Internal error while calling method. Please try again later"} - 5064 ms
2013-08-09 15:42:33 -0400 DEBUG: passed in token: AQIC5wM2LY4SfcwVdqRaz6bcq4j7zP0hAxEZCHypVI1vmFM=@AAJTSQACMDMAAlNLAAotNTkxMTY2MTczAAJTMQACMDE=#
2013-08-09 15:42:33 -0400 DEBUG: cache key: business-entity-AQIC5wM2LY4SfcwVdqRaz6bcq4j7zP0hAxEZCHypVI1vmFM=@AAJTSQACMDMAAlNLAAotNTkxMTY2MTczAAJTMQACMDE=#
2013-08-09 15:42:33 -0400 DEBUG: business entity not found in cache, making the call
2013-08-09 15:42:33 -0400 DEBUG: url for business entity list: http://localservices.carecloud.local:3000/business_entities/list_by_user.json?list_type=list&token=AQIC5wM2LY4SfcwVdqRaz6bcq4j7zP0hAxEZCHypVI1vmFM%3D%40AAJTSQACMDMAAlNLAAotNTkxMTY2MTczAAJTMQACMDE%3D%23
2013-08-09 15:42:38 -0400 ERROR: GET http://localservices.carecloud.local:3000/business_entities/list_by_user.json?list_type=list&token=AQIC5wM2LY4SfcwVdqRaz6bcq4j7zP0hAxEZCHypVI1vmFM%3D%40AAJTSQACMDMAAlNLAAotNTkxMTY2MTczAAJTMQACMDE%3D%23 FAILED - Connection refused - connect(2)
2013-08-09 15:42:38 -0400 WARN: ----127.0.0.1 "GET /v2/appointment/listbyid/2ae8b08d-5d41-40e5-b068-41803fc689c4?authentication=AQIC5wM2LY4SfcwVdqRaz6bcq4j7zP0hAxEZCHypVI1vmFM%3D%40AAJTSQACMDMAAlNLAAotNTkxMTY2MTczAAJTMQACMDE%3D%23" - 500 {"error":"Internal error while calling method. Please try again later"} - 5126 ms
2013-08-09 15:42:42 -0400 DEBUG: passed in token: AQIC5wM2LY4SfcwVdqRaz6bcq4j7zP0hAxEZCHypVI1vmFM=@AAJTSQACMDMAAlNLAAotNTkxMTY2MTczAAJTMQACMDE=#
2013-08-09 15:42:42 -0400 DEBUG: cache key: business-entity-AQIC5wM2LY4SfcwVdqRaz6bcq4j7zP0hAxEZCHypVI1vmFM=@AAJTSQACMDMAAlNLAAotNTkxMTY2MTczAAJTMQACMDE=#
2013-08-09 15:42:42 -0400 DEBUG: business entity not found in cache, making the call
2013-08-09 15:42:42 -0400 DEBUG: url for business entity list: http://localservices.carecloud.local:3000/business_entities/list_by_user.json?list_type=list&token=AQIC5wM2LY4SfcwVdqRaz6bcq4j7zP0hAxEZCHypVI1vmFM%3D%40AAJTSQACMDMAAlNLAAotNTkxMTY2MTczAAJTMQACMDE%3D%23
2013-08-09 15:42:47 -0400 ERROR: GET http://localservices.carecloud.local:3000/business_entities/list_by_user.json?list_type=list&token=AQIC5wM2LY4SfcwVdqRaz6bcq4j7zP0hAxEZCHypVI1vmFM%3D%40AAJTSQACMDMAAlNLAAotNTkxMTY2MTczAAJTMQACMDE%3D%23 FAILED - Connection refused - connect(2)
2013-08-09 15:42:47 -0400 WARN: ----127.0.0.1 "GET /v1/appointment/listbyid/2ae8b08d-5d41-40e5-b068-41803fc689c4?authentication=AQIC5wM2LY4SfcwVdqRaz6bcq4j7zP0hAxEZCHypVI1vmFM%3D%40AAJTSQACMDMAAlNLAAotNTkxMTY2MTczAAJTMQACMDE%3D%23" - 500 {"error":"Internal error while calling method. Please try again later"} - 5070 ms
2013-08-09 15:42:56 -0400 DEBUG: passed in token: AQIC5wM2LY4SfcwVdqRaz6bcq4j7zP0hAxEZCHypVI1vmFM=@AAJTSQACMDMAAlNLAAotNTkxMTY2MTczAAJTMQACMDE=#
2013-08-09 15:42:56 -0400 DEBUG: cache key: business-entity-AQIC5wM2LY4SfcwVdqRaz6bcq4j7zP0hAxEZCHypVI1vmFM=@AAJTSQACMDMAAlNLAAotNTkxMTY2MTczAAJTMQACMDE=#
2013-08-09 15:42:56 -0400 DEBUG: business entity not found in cache, making the call
2013-08-09 15:42:56 -0400 DEBUG: url for business entity list: http://localservices.carecloud.local:3000/business_entities/list_by_user.json?list_type=list&token=AQIC5wM2LY4SfcwVdqRaz6bcq4j7zP0hAxEZCHypVI1vmFM%3D%40AAJTSQACMDMAAlNLAAotNTkxMTY2MTczAAJTMQACMDE%3D%23
2013-08-09 15:43:01 -0400 ERROR: GET http://localservices.carecloud.local:3000/business_entities/list_by_user.json?list_type=list&token=AQIC5wM2LY4SfcwVdqRaz6bcq4j7zP0hAxEZCHypVI1vmFM%3D%40AAJTSQACMDMAAlNLAAotNTkxMTY2MTczAAJTMQACMDE%3D%23 FAILED - Connection refused - connect(2)
2013-08-09 15:43:01 -0400 WARN: ----127.0.0.1 "GET /v2/appointment/listbyid/2ae8b08d-5d41-40e5-b068-41803fc689c4?authentication=AQIC5wM2LY4SfcwVdqRaz6bcq4j7zP0hAxEZCHypVI1vmFM%3D%40AAJTSQACMDMAAlNLAAotNTkxMTY2MTczAAJTMQACMDE%3D%23" - 500 {"error":"Internal error while calling method. Please try again later"} - 5128 ms
2013-08-09 15:45:31 -0400 DEBUG: passed in token: AQIC5wM2LY4SfcwVdqRaz6bcq4j7zP0hAxEZCHypVI1vmFM=@AAJTSQACMDMAAlNLAAotNTkxMTY2MTczAAJTMQACMDE=#
2013-08-09 15:45:31 -0400 DEBUG: cache key: business-entity-AQIC5wM2LY4SfcwVdqRaz6bcq4j7zP0hAxEZCHypVI1vmFM=@AAJTSQACMDMAAlNLAAotNTkxMTY2MTczAAJTMQACMDE=#
2013-08-09 15:45:31 -0400 DEBUG: business entity not found in cache, making the call
2013-08-09 15:45:31 -0400 DEBUG: url for business entity list: http://localservices.carecloud.local:3000/business_entities/list_by_user.json?list_type=list&token=AQIC5wM2LY4SfcwVdqRaz6bcq4j7zP0hAxEZCHypVI1vmFM%3D%40AAJTSQACMDMAAlNLAAotNTkxMTY2MTczAAJTMQACMDE%3D%23
2013-08-09 15:45:36 -0400 ERROR: GET http://localservices.carecloud.local:3000/business_entities/list_by_user.json?list_type=list&token=AQIC5wM2LY4SfcwVdqRaz6bcq4j7zP0hAxEZCHypVI1vmFM%3D%40AAJTSQACMDMAAlNLAAotNTkxMTY2MTczAAJTMQACMDE%3D%23 FAILED - Connection refused - connect(2)
2013-08-09 15:45:36 -0400 WARN: ----127.0.0.1 "GET /v2/appointment/listbyid/55f3fa84-8fe9-481d-a587-5c4c082333e7?authentication=AQIC5wM2LY4SfcwVdqRaz6bcq4j7zP0hAxEZCHypVI1vmFM%3D%40AAJTSQACMDMAAlNLAAotNTkxMTY2MTczAAJTMQACMDE%3D%23" - 500 {"error":"Internal error while calling method. Please try again later"} - 5067 ms
2013-08-09 15:46:31 -0400 DEBUG: passed in token: AQIC5wM2LY4SfcwVdqRaz6bcq4j7zP0hAxEZCHypVI1vmFM=@AAJTSQACMDMAAlNLAAotNTkxMTY2MTczAAJTMQACMDE=#
2013-08-09 15:46:31 -0400 DEBUG: cache key: business-entity-AQIC5wM2LY4SfcwVdqRaz6bcq4j7zP0hAxEZCHypVI1vmFM=@AAJTSQACMDMAAlNLAAotNTkxMTY2MTczAAJTMQACMDE=#
2013-08-09 15:46:31 -0400 DEBUG: business entity not found in cache, making the call
2013-08-09 15:46:31 -0400 DEBUG: url for business entity list: http://localservices.carecloud.local:3000/business_entities/list_by_user.json?list_type=list&token=AQIC5wM2LY4SfcwVdqRaz6bcq4j7zP0hAxEZCHypVI1vmFM%3D%40AAJTSQACMDMAAlNLAAotNTkxMTY2MTczAAJTMQACMDE%3D%23
2013-08-09 15:46:36 -0400 ERROR: GET http://localservices.carecloud.local:3000/business_entities/list_by_user.json?list_type=list&token=AQIC5wM2LY4SfcwVdqRaz6bcq4j7zP0hAxEZCHypVI1vmFM%3D%40AAJTSQACMDMAAlNLAAotNTkxMTY2MTczAAJTMQACMDE%3D%23 FAILED - Connection refused - connect(2)
2013-08-09 15:46:36 -0400 WARN: ----127.0.0.1 "GET /v2/appointment/listbyid/55f3fa84-8fe9-481d-a587-5c4c082333e7?authentication=AQIC5wM2LY4SfcwVdqRaz6bcq4j7zP0hAxEZCHypVI1vmFM%3D%40AAJTSQACMDMAAlNLAAotNTkxMTY2MTczAAJTMQACMDE%3D%23" - 500 {"error":"Internal error while calling method. Please try again later"} - 5070 ms
2013-08-09 15:48:10 -0400 DEBUG: passed in token: AQIC5wM2LY4SfcwVdqRaz6bcq4j7zP0hAxEZCHypVI1vmFM=@AAJTSQACMDMAAlNLAAotNTkxMTY2MTczAAJTMQACMDE=#
2013-08-09 15:48:10 -0400 DEBUG: cache key: business-entity-AQIC5wM2LY4SfcwVdqRaz6bcq4j7zP0hAxEZCHypVI1vmFM=@AAJTSQACMDMAAlNLAAotNTkxMTY2MTczAAJTMQACMDE=#
2013-08-09 15:48:10 -0400 DEBUG: business entity not found in cache, making the call
2013-08-09 15:48:10 -0400 DEBUG: url for business entity list: http://localservices.carecloud.local:3000/business_entities/list_by_user.json?list_type=list&token=AQIC5wM2LY4SfcwVdqRaz6bcq4j7zP0hAxEZCHypVI1vmFM%3D%40AAJTSQACMDMAAlNLAAotNTkxMTY2MTczAAJTMQACMDE%3D%23
2013-08-09 15:48:15 -0400 ERROR: GET http://localservices.carecloud.local:3000/business_entities/list_by_user.json?list_type=list&token=AQIC5wM2LY4SfcwVdqRaz6bcq4j7zP0hAxEZCHypVI1vmFM%3D%40AAJTSQACMDMAAlNLAAotNTkxMTY2MTczAAJTMQACMDE%3D%23 FAILED - Connection refused - connect(2)
2013-08-09 15:48:15 -0400 WARN: ----127.0.0.1 "GET /v2/appointment/listbyid/55f3fa84-8fe9-481d-a587-5c4c082333e7?authentication=AQIC5wM2LY4SfcwVdqRaz6bcq4j7zP0hAxEZCHypVI1vmFM%3D%40AAJTSQACMDMAAlNLAAotNTkxMTY2MTczAAJTMQACMDE%3D%23" - 500 {"error":"Internal error while calling method. Please try again later"} - 5474 ms
2013-08-09 15:48:55 -0400 DEBUG: in base
2013-08-09 15:48:55 -0400 INFO: ----127.0.0.1 "GET /" - 200  - 0 ms
2013-08-09 15:48:55 -0400 WARN: ----127.0.0.1 "GET /favicon.ico" - 404  - 0 ms
2013-08-09 15:49:07 -0400 DEBUG: in base
2013-08-09 15:49:07 -0400 INFO: ----127.0.0.1 "GET /" - 200  - 0 ms
2013-08-09 15:49:20 -0400 DEBUG: passed in token: AQIC5wM2LY4SfcwVdqRaz6bcq4j7zP0hAxEZCHypVI1vmFM=@AAJTSQACMDMAAlNLAAotNTkxMTY2MTczAAJTMQACMDE=#
2013-08-09 15:49:20 -0400 DEBUG: cache key: business-entity-AQIC5wM2LY4SfcwVdqRaz6bcq4j7zP0hAxEZCHypVI1vmFM=@AAJTSQACMDMAAlNLAAotNTkxMTY2MTczAAJTMQACMDE=#
2013-08-09 15:49:20 -0400 DEBUG: business entity not found in cache, making the call
2013-08-09 15:49:20 -0400 DEBUG: url for business entity list: http://localservices.carecloud.local:3000/business_entities/list_by_user.json?list_type=list&token=AQIC5wM2LY4SfcwVdqRaz6bcq4j7zP0hAxEZCHypVI1vmFM%3D%40AAJTSQACMDMAAlNLAAotNTkxMTY2MTczAAJTMQACMDE%3D%23
2013-08-09 15:49:25 -0400 ERROR: GET http://localservices.carecloud.local:3000/business_entities/list_by_user.json?list_type=list&token=AQIC5wM2LY4SfcwVdqRaz6bcq4j7zP0hAxEZCHypVI1vmFM%3D%40AAJTSQACMDMAAlNLAAotNTkxMTY2MTczAAJTMQACMDE%3D%23 FAILED - Connection refused - connect(2)
2013-08-09 15:49:25 -0400 WARN: ----127.0.0.1 "GET /v2/appointment/listbyid/55f3fa84-8fe9-481d-a587-5c4c082333e7?authentication=AQIC5wM2LY4SfcwVdqRaz6bcq4j7zP0hAxEZCHypVI1vmFM%3D%40AAJTSQACMDMAAlNLAAotNTkxMTY2MTczAAJTMQACMDE%3D%23" - 500 {"error":"Internal error while calling method. Please try again later"} - 5113 ms
2013-08-09 15:52:34 -0400 DEBUG: passed in token: AQIC5wM2LY4SfcwVdqRaz6bcq4j7zP0hAxEZCHypVI1vmFM=@AAJTSQACMDMAAlNLAAotNTkxMTY2MTczAAJTMQACMDE=#
2013-08-09 15:52:34 -0400 DEBUG: cache key: business-entity-AQIC5wM2LY4SfcwVdqRaz6bcq4j7zP0hAxEZCHypVI1vmFM=@AAJTSQACMDMAAlNLAAotNTkxMTY2MTczAAJTMQACMDE=#
2013-08-09 15:52:34 -0400 DEBUG: business entity not found in cache, making the call
2013-08-09 15:52:34 -0400 DEBUG: url for business entity list: http://localservices.carecloud.local:3000/business_entities/list_by_user.json?list_type=list&token=AQIC5wM2LY4SfcwVdqRaz6bcq4j7zP0hAxEZCHypVI1vmFM%3D%40AAJTSQACMDMAAlNLAAotNTkxMTY2MTczAAJTMQACMDE%3D%23
2013-08-09 15:52:39 -0400 ERROR: GET http://localservices.carecloud.local:3000/business_entities/list_by_user.json?list_type=list&token=AQIC5wM2LY4SfcwVdqRaz6bcq4j7zP0hAxEZCHypVI1vmFM%3D%40AAJTSQACMDMAAlNLAAotNTkxMTY2MTczAAJTMQACMDE%3D%23 FAILED - Connection refused - connect(2)
2013-08-09 15:52:39 -0400 WARN: ----127.0.0.1 "GET /v2/appointment/listbyid/484cf9ca-eed8-4b28-989f-5971d31d34e1?authentication=AQIC5wM2LY4SfcwVdqRaz6bcq4j7zP0hAxEZCHypVI1vmFM%3D%40AAJTSQACMDMAAlNLAAotNTkxMTY2MTczAAJTMQACMDE%3D%23" - 500 {"error":"Internal error while calling method. Please try again later"} - 5063 ms
2013-08-09 15:53:36 -0400 DEBUG: passed in token: AQIC5wM2LY4SfcwVdqRaz6bcq4j7zP0hAxEZCHypVI1vmFM=@AAJTSQACMDMAAlNLAAotNTkxMTY2MTczAAJTMQACMDE=#
2013-08-09 15:53:36 -0400 DEBUG: cache key: business-entity-AQIC5wM2LY4SfcwVdqRaz6bcq4j7zP0hAxEZCHypVI1vmFM=@AAJTSQACMDMAAlNLAAotNTkxMTY2MTczAAJTMQACMDE=#
2013-08-09 15:53:36 -0400 DEBUG: business entity not found in cache, making the call
2013-08-09 15:53:36 -0400 DEBUG: url for business entity list: http://localservices.carecloud.local:3000/business_entities/list_by_user.json?list_type=list&token=AQIC5wM2LY4SfcwVdqRaz6bcq4j7zP0hAxEZCHypVI1vmFM%3D%40AAJTSQACMDMAAlNLAAotNTkxMTY2MTczAAJTMQACMDE%3D%23
2013-08-09 15:53:49 -0400 DEBUG: {"business_entities":[{"id":1,"parent_id":7,"name":"ALBERTO DOMINGUEZ BALI","photo_handler":"","photo":{},"user_profile_id":219096,"permissions":{"can_access":true},"type":"SA"}],"recently_visited":[1]}
2013-08-09 15:53:49 -0400 DEBUG: returned business entity id: 1
2013-08-09 15:53:49 -0400 DEBUG: ++++++++++cache set
2013-08-09 15:53:49 -0400 DEBUG: url for appointment: http://localservices.carecloud.local:3000/appointments/1/484cf9ca-eed8-4b28-989f-5971d31d34e1/listbyexternalid.json?token=AQIC5wM2LY4SfcwVdqRaz6bcq4j7zP0hAxEZCHypVI1vmFM%3D%40AAJTSQACMDMAAlNLAAotNTkxMTY2MTczAAJTMQACMDE%3D%23
2013-08-09 15:53:54 -0400 DEBUG: [{"p_ext_id":"ef83b311-3db1-47e7-80ba-96b0e14f5ad4","p_id":"2645619","id":"4212600","start_time":"2013-05-29 10:00:00","end_time":"2013-05-29 10:15:00","location_id":"2","nature_of_visit_id":"17","appointment_status_id":"6","is_confirmed":"f","appointment_cancellation_reason_id":null,"cancellation_comments":null,"admit_diagnosis_id":null,"note_set_id":"37734989","document_set_id":"42769384","resource_id":"862","confirmation_details":null,"cancellation_details":null,"business_entity_id":"1","contrast":null,"laterality":null,"recurrence_id":null,"created_at":"2013-04-29 11:37:28.480712","updated_at":"2013-08-09 11:50:52.893597","created_by":"113","updated_by":"13389","arrived_at":"2013-05-29 10:23:40.010176","recurrence_index":null,"reason_for_visit":null,"task_id":null,"departed_at":null,"exam_room_id":null,"external_id":"484cf9ca-eed8-4b28-989f-5971d31d34e1"}]
2013-08-09 15:53:54 -0400 DEBUG: 484cf9ca-eed8-4b28-989f-5971d31d34e1
2013-08-09 15:53:54 -0400 DEBUG: 484cf9ca-eed8-4b28-989f-5971d31d34e1
2013-08-09 15:53:54 -0400 DEBUG: ef83b311-3db1-47e7-80ba-96b0e14f5ad4
2013-08-09 15:53:54 -0400 DEBUG: url for patient: http://localservices.carecloud.local:3000/businesses/1/patients/ef83b311-3db1-47e7-80ba-96b0e14f5ad4/externalid.json?token=AQIC5wM2LY4SfcwVdqRaz6bcq4j7zP0hAxEZCHypVI1vmFM%3D%40AAJTSQACMDMAAlNLAAotNTkxMTY2MTczAAJTMQACMDE%3D%23&do_full_export=true
2013-08-09 15:54:08 -0400 DEBUG: #<Net::HTTPOK:0x007f903f81bab0>
2013-08-09 15:54:08 -0400 INFO: ----127.0.0.1 "GET /v2/appointment/listbyid/484cf9ca-eed8-4b28-989f-5971d31d34e1?authentication=AQIC5wM2LY4SfcwVdqRaz6bcq4j7zP0hAxEZCHypVI1vmFM%3D%40AAJTSQACMDMAAlNLAAotNTkxMTY2MTczAAJTMQACMDE%3D%23" - 200  - 32068 ms
